debian/indi-avalon/changelog

```diff
@@ -1,5 +1,46 @@
-indi-avalon (1.0) bionic; urgency=medium
+indi-avalon (1.3) bionic; urgency=medium
```

Comment:
This is an invalid changelog .. the debian one is used to build packages and have very specific format. Please duplicate my first entry and modify it. or use **dch** to append a new changelog. We have a problem just yesterday with indi-starbook failing to build for a couple of weeks and it was traced to a _single space_ that wasn't added!